app/Http/Controllers/DepartmentController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Lookup\Department;
+
+class DepartmentController extends Controller
+{
+    /**
+     * Return all departments as an array
+     *
+     * @return mixed
+     */
+    public function index()
```

Comment:
If we're adding this controller, we might as well create an api route while we're at it.